index.html

```diff
@@ -0,0 +1,45 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <link rel="stylesheet" href="style.css">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Poppins:wght@400;700&display=swap" rel="stylesheet">
+    <title>Portfolio</title>
+</head>
+<body>
+    <main>
+        
+            <nav>
+                <div class="logo">
+                    <p>My Logo</p>
+                </div>
+                
+                <div class="menu-icon">
+                    <img src="images/menu-icon.svg" alt="">
+                </div>
+                
+              
+            </nav>
+        <section class="first-section">
+            <header>
```

Comment:
- [ ] Please remove the `<header>` tag from this section to make sure the code follows best practices and is sematic. The `<header>` tag should only be responsible for the toolbar (or header) section of the page.